src/Platforms/AbstractPlatform.php

```diff
@@ -244,10 +244,13 @@ public function getEnumDeclarationSQL(array $column): string
             throw ColumnValuesRequired::new($this, 'ENUM');
         }
 
-        $length = count($column['values']) > 1
+        $configuredLength = $column['length'] ?? 0;
+        $neededLength     = count($column['values']) > 1
             ? max(...array_map(mb_strlen(...), $column['values']))
             : mb_strlen($column['values'][key($column['values'])]);
 
+        $length = max($configuredLength, $neededLength);
```

Comment:
So, if we explicitly configure a length that is too short to store the longest enum value, we silently override the explicitly configured value? 🤔

Reply:
You can throw now already.

Reply:
Done